pkgs/applications/networking/instant-messengers/pantalaimon/default.nix

```
@@ -61,6 +62,10 @@ python3Packages.buildPythonApplication rec {
   ]
   ++ lib.flatten (lib.attrValues optional-dependencies);
 
+  nativeBuildInputs = lib.optionals enableDbusUi [
+    wrapGAppsHook
+  ];
```

Also this will double-wrap, you need

```suggestion
  nativeBuildInputs = lib.optionals enableDbusUi [
    wrapGAppsHook3
  ];
  
  dontWrapGApps = true;
  makeWrapperArgs = [
    "\${gappsWrapperArgs[@]}"
  ];
```

---

well optional it on the dbus bool, but you get it

---

done